data/json/overmap_specials.json

```diff
@@ -220,6 +220,20 @@
         "flags" : [ "CLASSIC" ]
     },{
         "type" : "overmap_special",
+        "id" : "Sugar House",
+        "overmaps" : [
+            { "point":[0,0,0], "overmap": "sugar_house" }
```

Comment:
> Should I simply change this overmap value from sugar_house to sugar_house_north

Yes, precisely.

>  Will I need to change it somewhere else?

No, that's the only change that needs to be made. Since this overmap terrain rotates, there are:
`"sugar_house_north"`, `"sugar_house_east"`, `"sugar_house_south"`, and `"sugar_house_west"`, but no `"sugar_house"`.